src/pages/rootpages/AllProducts.tsx

```diff
@@ -1,12 +1,74 @@
-import ProductLoader from '../../components/rootcomponents/ProductLoader';
+import React from 'react';
+import { AllAvailableProducts } from '../../components/rootcomponents/allAvailableProducts';
+import Footer from '../../components/rootcomponents/Footer';
 
-const AllProducts = () => {
+export const AllProducts: React.FC = () => {
   return (
-    <div className="m-8">
-      <h1>All Products</h1>
-      <ProductLoader count={9} />
+    <div>
+      <div className="flex mx-10 mt-12 mb-4 gap-6 text-xl font-medium md:gap-0">
+        <div className="w-1/4 pl-6 hidden md:block md:justify-center md:">
+          Categories
+        </div>
+        <div className="w-3/4 pl-6">Explore All Products</div>
+      </div>
+
+      <div className="flex mx-10 mb-12 gap-6">
+        <div className="w-1/4 h-96 items-center bg-slate-100 shadow-md px-6 pt-6 pb-8 text-gray-500 rounded-md hidden md:block">
+          <p className="text-lg mb-2">Categories</p>
+          <form>
+            <div className="mb-1">
+              <label className="inline-flex items-center cursor-pointer">
+                <input
+                  type="radio"
+                  className="form-radio h-4 w-4 text-blue-600"
+                  name="radio"
+                  value="option1"
+                />
+                <span className="ml-2">Home & Living</span>
+              </label>
+            </div>
+            <div className="mb-1">
+              <label className="inline-flex items-center cursor-pointer">
+                <input
+                  type="radio"
+                  className="form-radio h-4 w-4 text-blue-600"
+                  name="radio"
+                  value="option2"
+                />
+                <span className="ml-2">Technology</span>
+              </label>
+            </div>
+            <div className="mb-1">
+              <label className="inline-flex items-center cursor-pointer">
+                <input
+                  type="radio"
+                  className="form-radio h-4 w-4 text-blue-600"
+                  name="radio"
+                  value="option3"
+                />
+                <span className="ml-2">Clothing & Shoes</span>
+              </label>
+            </div>
+            <div className="mb-1">
+              <label className="inline-flex items-center cursor-pointer">
+                <input
+                  type="radio"
+                  className="form-radio h-4 w-4 text-blue-600"
+                  name="radio"
+                  value="option3"
+                />
+                <span className="ml-2">Fashion Design</span>
+              </label>
+            </div>
```

Comment:
This is React, Why can't you make this a component? Why are you repeating yourself???